### ISSUE TITLE
$ref errors displayed for staged changes

### ISSUE THREAD

**Issue by smcx24** [42Crunch/vscode-openapi#179]
Steps:

1. Add file to staging that contains a correct $ref to a component in another file 
2. Select the staged file to show the file changes
3. It will display an error for the $ref as it tries to reference the external file using the 'git:/' protocol
4. Also, if you close the 'file changes' window, the errors do not disappear from the Problems panel

```
Error reading file "git:/c%3A/Users/userAcc/source/repos/apispec/spec.yml: cannot open git:/c%3A/Users/userAcc/source/repos/apispec/spec.yml. 
Detail: Unable to read file 'git:/c:/Users/userAcc/source/repos/apispec/spec.yml' (SyntaxError: Unexpected end of JSON input)"
```

**Comment by Ghostwoods** [comment]
I had a very similar problem today; reverting VSCode's YAML extension to v1.4.0 fixed it. There may be more recent versions that also work.

**Comment by ak1394** [comment]
Thanks for reporting the issue guys, we're getting ready to release a new update in a few days, and I'll look into fixing this right after that.

**Comment by ak1394** [comment]
Fixed in 4.13.0
